src/pages/messages/messages.html

```diff
@@ -19,6 +19,11 @@
       <div *ngFor="let message of day.messages" class="message-wrapper">
         <div [class]="'message message-' + message.ownership">
           <div *ngIf="message.type == 'text'" class="message-content message-content-text">{{message.content}}</div>
+          <div *ngIf="message.type == 'location'" class="message-content message-content-text">
+            <sebm-google-map [zoom]="getLocation(message.content).zoom" [latitude]="getLocation(message.content).lat" [longitude]="getLocation(message.content).lng">
```

Comment:
I thought about it but it's already an external component which in use only in one place... if it were a real app, then we would split it, but I don't want it to be too compilcated.